home page edits

johanvillatoro opened the pull request:
Finished home page, side tabs, finalized webr code.

critmcdonald commented:
Some thoughts:

## Index

- given our aim is to remove learnr, we could remove that once the training exercises are converted.
- We need to make it clear that these lessons can be done online with these pages. We note they can be downloaded, but let's make a bigger deal that they can be worked online.
- let's rearrange the Data Sources bit. Instead of this big list, we'll want to explain at the top that these lessons are all built on a star wars theme and try to use the characters data as much as possible. This list of data sources could instead be moved either to an about page or the JedR Contribution piece.
- Be sure to add your name to the credits at the bottom!
- In the Overview let's list the Trials before the training since those will likely be used more.

> I have more to add later.

critmcdonald commented:
comments and suggestions:

> More to come ... just saving for now.

- [ ] let's move the webr files into the `trials/` and `training/` folders and move the old ones into some kind of archive folder. That will also allow us to set it not to process. We'll have to fix links, too.
- [ ] Small thing, but let's see if we can get the trail/training name in the page title when it is running.
- [ ] I recognize we don't have the same kind of quiz tool in Quarto Live that we had with learnr, but I'm not in favor of entering the answer as code into a code block. For now, let's make them a rhetorical question and set them out in some kind of callout box. After that we can maybe look for some kind of external tool were we could embed a poll. Or some other R package to do one. I bet there is one. This is also hampered for the time being that we can't always see the result the way the way because of how results are printed.
- [ ] When you leave a blank ___ for an answer, leave the correct number of _ per character. So `______(______)` for `arrange(height)`
- [ ] Is there a way to connect one trial to the next at the bottom? It could be that we reconfigure the project as a book instead of a website? But there might be a less destructive way to do that.

Intro
- [ ] Focus on variables doesn't have a solution for arrange.

johanvillatoro commented:
i only saw half of it from my email. So, I'm not completely done...

johanvillatoro commented:
comments and suggestions:
- Used .gitignore to ignore archive folder (check to make sure its done right)
- Hint cannot be displayed between the code and the result, but there is another possible solution to it.
- connected all the pages with one another at the bottom of each page
- all "quiz" question reworked using `collapase

Intro:
- I have an example of a tabbed hint. Maybe it fixes the problem (view intro first question)
- I added underscores to all code chucks. Maybe this way we can give the user an outline but not the solution. Let me know if you want me to change.

Count:
- Again, I added the underscores to all the trials. If you like it for the count section we can keep it if not, I can add the group_by/summarize/arrange to the same quest/chunk.

Math:
- "If you attempt to match rows containing NA values, the result will also be NA by default. To address this, you can use na.rm = TRUE to exclude rows with NA values from the operation." better?
- "Don't forget that we often want to sort the `height_avg` by **desc**ending order, because journalists are typically interested in the *most* of something." reusing previous wording to specify using desc()

Plot:
- I think we should work on this together in person.

Pivot:
- used "echo: false" to hide code. It takes time to load in, but like the force... you can't see it

Join:
- using kable() from `knitr` allows everything to show. Nice steps foward, but might not be solution. Use this code to see if for yourself.
- "characters <- droids |> 
  bind_rows(humans, other) |>
  kable()

characters"
